hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -50,7 +53,8 @@
     KAFKA_CONSUMER_METRICS_NUM_SAMPLES_CONFIG("kafka.consumer.metrics.num.samples", 2),
     KAFKA_CONSUMER_REQUEST_TIMEOUT_MS_CONFIG("kafka.consumer.request.timeout.ms", 250_000),
     KAFKA_CONSUMER_CONNECTIONS_MAX_IDLE_MS_CONFIG("kafka.consumer.connections.max.idle.ms", 9 * 60 * 1000),
-    KAFKA_CONSUMER_MAX_POLL_RECORDS_CONFIG("kafka.consumer.max.poll.records", 10),
+    KAFKA_CONSUMER_MAX_POLL_RECORDS_CONFIG("kafka.consumer.max.poll.records", 1),
+    KAFKA_CONSUMER_USE_GLOBAL_MAX_PARTITION_FETCH("kafka.consumer.global.max.partition.fetch", true),
```

Comment:
i would rather have positive flag (turn this on) that negative (turn this off). besides this is hermes consumer flag, not kafka consumer. maybe `CONSUMER_USE_TOPIC_MESSAGE_SIZE`?